docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,479 @@
+ 
+* Create the **VPC** using the provided YAML and `kubectl apply`:
+```
+cat <<EOF > vpc.yaml
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: VPC
+metadata:
+  name: vpc-tutorial-test
+spec:
+  cidrBlocks: 
+  - 10.0.0.0/16
+  enableDNSSupport: true
+  enableDNSHostnames: true
+EOF
+ 
+kubectl apply -f vpc.yaml
+```
+ 
+* Check the **VPC** `Status` using `kubectl describe`:
+```
+> kubectl describe vpcs
+...
+Status:
+  Ack Resource Metadata:
+    Owner Account ID:  <ID>
+    Region:            us-west-2
+  Cidr Block Association Set:
+    Association ID:  vpc-cidr-assoc-<ID>
+    Cidr Block:      10.0.0.0/16
+    Cidr Block State:
+      State:  associated
+  Conditions:
+    Last Transition Time:  2022-10-12T17:26:08Z
+    Message:               Resource synced successfully
+    Reason:
+    Status:                True
+    Type:                  ACK.ResourceSynced
+  Dhcp Options ID:         dopt-<ID>
+  Is Default:              false
+  Owner ID:                <ID>
+  State:                   available
+  Vpc ID:                  vpc-<ID>
+Events:                    <none>
+```
+ 
+* The **VPC** resource synced successfully and is available. Note the `vpc-<ID>`.
+ 
+* Create the **Subnet** using `vpc-<ID>`, the provided YAML, and `kubectl apply`:
+```
+cat <<EOF > subnet.yaml
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: subnet-tutorial-test
+spec:
+  cidrBlock: 10.0.0.0/20
+  vpcID: vpc-<ID>
+EOF
+ 
+kubectl apply -f subnet.yaml
+```
+ 
+* Check the **Subnet** availability and ID using `kubectl describe`:
+```
+> kubectl describe subnets
+...
+Status:
+  Ack Resource Metadata:
+    Arn:                       arn:aws:ec2:us-west-2:<ID>:subnet/subnet-<ID>
+    Owner Account ID:          <ID>
+    Region:                    us-west-2
+  Available IP Address Count:  4091
+  Conditions:
+    Last Transition Time:           2022-10-12T17:36:53Z
+    Message:                        Resource synced successfully
+    Reason:
+    Status:                         True
+    Type:                           ACK.ResourceSynced
+  Default For AZ:                   false
+  Map Customer Owned IP On Launch:  false
+  Owner ID:                         <ID>
+  Private DNS Name Options On Launch:
+  State:      available
+  Subnet ID:  subnet-<ID>
+Events:       <none>
+```
+ 
+* Delete the resources:
+  * `kubectl delete -f subnet.yaml`
+  * `kubectl delete -f vpc.yaml`
+ 
+ 
+Both resources were successfully deployed, managed, then deleted by their respective controllers. Although contrived, this example highlights how easy it can be to deploy AWS resources via YAML files and how it feels like managing any other K8s resource. 
+ 
+In this example, waiting for the `vpcID` to be generated and manually updating the Subnet custom resource is not fully declarative. The next example alleviates these concerns using a single manifest (YAML) to deploy the entire network topology.
```

Comment:
```suggestion
In this example, we used multiple YAML manifests and waited for the `vpcID` to be generated before manually updating the Subnet custom resources to reference that VPC ID. This technique is less aligned to a fully declarative, GitOps-style of configuration management because dependencies between resources need to be manually resolved using a specific order of operations. The next example uses a single YAML manifest to deploy an entire network topology using ACK Resource References, a technique that better aligns with a fully automated and declarative GitOps-style of configuration management.
```